src/main/java/seedu/weme/model/Weme.java

```diff
@@ -233,6 +233,7 @@ public boolean isStagingAreaEmpty() {
     public void addMeme(Meme m) {
         memes.add(m);
         stats.addDefaultLikeData(m);
+        stats.addDefaultDislikeData(m);
```

Comment:
Can we add stats first before adding the meme? Adding the meme first causes error messages to appear in the console as gridview tries to refresh immediately after the change